connector/signaltometricsconnector/factory.go

```diff
@@ -34,6 +34,8 @@ import (
 	"github.com/elastic/opentelemetry-collector-components/connector/signaltometricsconnector/internal/model"
 )
 
+const collectorInstanceResourceAttributePrefix = "observer"
```

Comment:
I think this should also be configurable since there is no convention/standard around such use-cases where single writer is violated due to removing resource attribute and then restored by adding otel collector's instance information.

Reply:
Updated!